postpic/datahandling.py

```diff
@@ -1867,7 +1867,8 @@ def saveto(self, filename):
         self.export(filename)
 
     def __str__(self):
-        return '<Feld "' + self.name + '" ' + str(self.shape) + '>'
+        s = '<Field "{:}" {:}>'
+        return s.format(str(self.name), str(self.shape))
```

Comment:
using explicit `str()` conversion kind of defeats the purpose of `.format()`.

```
return '<Field "{}" {}>'.format(self.name, self.shape)
```

should be enough. Using nothing as a format makes `format()` fallback to `str()` in most cases, anyway. If self.name should be something other than a string at some later time, it should provide a `__format__` method.